### PR TITLE
Add: crop csv file and import to the database

### PR DESCRIPTION
**Pull request by sigu**
Get crops from the CSV file and add them to the database
Refers to issue #359

### CODE REVIEW

File: lib/tasks/import_crops.rake [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    crops = CSV.read("#{Rails.root}/lib/crops.csv")
+    # remove nil or empty names, get unique names then save the crop
+    crops.reject { |crop| crop[0].nil? || crop[0].empty? }
+         .uniq { |crop| crop[0].downcase }
+         .each { |crop| save_crop(crop) }
+  end
+
+  def save_crop(crop)
+    return if Crop.where(binomial_name: crop.first).exists?
+    Crop.create!(name: crop[0], binomial_name: crop[0], description: crop[1])
```
Comment (simonv3) [inline lib/tasks/import_crops.rake]: In response  @roryaronson's comments in the issue, this looks like it can be updated?

**Comment by sigu** [comment]
@simonv3  is there any documentation that needs to be modified to show why and how this import is done.

**Comment by simonv3** [comment]
No but feel free to add some!

**Comment by sigu** [comment]
Not any documentation adjustment I can think of right now. 😄

**Comment by simonv3** [comment]
Merging cause I think the test failures are outside of this PR.

**Pull request merged**